From 8ef641c89e3c2ff303eb362a7b77760dd2ed7e69 Mon Sep 17 00:00:00 2001
From: "Dutta, Ranjan" <ranjan.dutta@intel.com>
Date: Tue, 10 Mar 2020 22:04:27 +0800
Subject: [PATCH 3/3] Revert "driver core: Print device when resources present
 in really_probe()"

This reverts commit 73504f095df344ad50be94c760d7c6188ad68c15.

Tracked-On: PKT-3178
---
 drivers/base/dd.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

Index: kernel-lts-staging/drivers/base/dd.c
===================================================================
--- kernel-lts-staging.orig/drivers/base/dd.c
+++ kernel-lts-staging/drivers/base/dd.c
@@ -470,11 +470,7 @@ static int really_probe(struct device *d
 	atomic_inc(&probe_count);
 	pr_debug("bus: '%s': %s: probing driver %s with device %s\n",
 		 drv->bus->name, __func__, drv->name, dev_name(dev));
-	if (!list_empty(&dev->devres_head)) {
-		dev_crit(dev, "Resources present before probing\n");
-		ret = -EBUSY;
-		goto done;
-	}
+	WARN_ON(!list_empty(&dev->devres_head));
 
 re_probe:
 	dev->driver = drv;
